<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!-- Start fake_bbox_angles node -->
  <node name="fake_bbox_angles" pkg="navigation" type="fake_bbox_angles_node" output="screen">
    <!-- Add any necessary arguments here -->
  </node>

  <!-- Start fake_robot_coords node -->
  <node name="fake_robot_coords" pkg="navigation" type="fake_robot_coords_node" output="screen">
    <!-- Add any necessary arguments here -->
  </node>

  <!-- Start fake_yolo_boxes node -->
  <node name="fake_yolo_boxes" pkg="navigation" type="fake_yolo_boxes_node" output="screen">
    <!-- Add any necessary arguments here -->
  </node>

  <!-- Start angle_finder node -->
  <node name="angle_finder" pkg="navigation" type="angle_finder_node" output="screen">
    <!-- Add any necessary arguments here -->
  </node>

  <!-- Start distance_finder node -->
  <node name="distance_finder" pkg="navigation" type="distance_finder_node" output="screen">
    <!-- Add any necessary arguments here -->
  </node>

  <!-- Start coord_finder node -->
  <node name="coord_finder" pkg="navigation" type="coord_finder_node" output="screen">
    <!-- Add any necessary arguments here -->
  </node>
</launch>